static/src/js/util/timeline.js

```diff
@@ -12,6 +13,49 @@ export const timelineIntervals = {
   decade: '5'
 }
 
+export const timelineZoomEnums = {
```

Comment:
Can we simply this to make it a little more intuitive.

Reply:
It looks like we have it this way because the values are coupled with what is expected inside of EDSC timeline.